lib/passport-azure-ad/saml.js

```diff
@@ -199,15 +201,15 @@ SAML.prototype.getLogoutUrl = function(req, callback) {
   this.requestToUrl(request, 'logout', callback);
 };
 
-SAML.prototype.validateSignature = function (xml, cert) {
+SAML.prototype.validateSignature = function(xml, cert) {
   var doc = new xmldom.DOMParser().parseFromString(xml);
   var signature = xmlCrypto.xpath(doc, "//*[local-name(.)='Signature' and namespace-uri(.)='http://www.w3.org/2000/09/xmldsig#']")[0];
   var sig = new xmlCrypto.SignedXml();
   sig.keyInfoProvider = {
-    getKeyInfo: function () {
+    getKeyInfo: function() {
       return "<X509Data></X509Data>";
     },
-    getKey: function () {
+    getKey: function() {
       //TODO: should I use the key in keyInfo or in cert?
```

Comment:
If you are using the cert in the token for check the signature, you must check that you trust that cert